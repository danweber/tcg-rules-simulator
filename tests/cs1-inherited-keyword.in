VERSION:2

#FIELD1:Attacker,CS1-03,DW1-Van Sourceless,CS1-03
FIELD1:Res,DW1-Van,CS1-Gideon Other,CS1-04,DW1-Van
FIELD2:Tank,CS2-12,REST
GAME:P1 T2 MAIN 5

TITLE:Custom Keyword from inherit
TEXT:Keyword comes from the inherit, will it be automatically applied?
DUMP:DP,KEYWORDS

ECHO: go
CMD:ATTACK Res Tank
STEP:AUTOANSWER:CAPTURE
DUMP:DP,KEYWORDS,TRASH


