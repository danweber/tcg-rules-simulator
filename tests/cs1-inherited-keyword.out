FIELD1:Res,DW1-Vanillamon,CS1-Gideontheguardian,5000DP,RESURRECTION Other,CS1-Stoneshield,DW1-Vanillamon,4500DP,RESURRECTION
FIELD2:Tank,CS2-Colossalsword,REST,7500DP
GAME:P1 T2 MAIN 5
ECHO: go
FIELD1:Other,CS1-Stoneshield,DW1-Vanillamon,4500DP,RESURRECTION I4,DW1-Vanillamon,5000DP
FIELD2:Tank,CS2-Colossalsword,REST,7500DP
TRASH1:CS1-Gideontheguardian
TRASH2:
GAME:P1 T2 MAIN 5
